templates/docs/examples/layouts/application-structure.html

```diff
@@ -2,12 +2,12 @@
 {% block title %}Application / Structure{% endblock %}
 
 {% block content %}
-<div class="l-application  app-demo">
+<div class="l-application app-demo" role="presentation">
   <div class="l-navigation-bar">
     <div class="u-clearfix"><code>l-navigation-bar</code> <button class="u-float-right js-menu-toggle is-dense u-no-margin">Menu</button></div>
   </div>
 
-  <div class="l-navigation">
+  <div class="l-navigation" role="banner">
```

Comment:
"By default, the HTML5 `<header>` element has an identical meaning to the banner landmark, unless it is a descendant of `<aside>`, `<article>`, `<main>`, `<nav>`, or `<section>`."

If we use HTML tags for other landmarks, maybe we should do the same for header/banner?

Reply:
```suggestion
  <header class="l-navigation">
```